autofaiss/external/build.py

```diff
@@ -137,7 +141,7 @@ def create_index(
         with Timeit("-> Extract training vectors", indent=2):
 
             memory_available_for_training = cast_bytes_to_memory_string(
-                cast_memory_to_bytes(current_memory_available) - metadata.estimated_index_size_in_bytes() * 0.25
+                cast_memory_to_bytes(current_memory_available) - 500_000_000
```

Comment:
isn't this now unneeded after your last PR ?

Reply:
You are rihgt. I removed it.